lib/studies/reply/bottom_drawer.dart

```diff
@@ -45,10 +47,9 @@ class BottomDrawer extends StatelessWidget {
               padding: const EdgeInsetsDirectional.only(start: 18),
               child: Text(
                 'FOLDERS',
-                style: Theme.of(context)
-                    .textTheme
-                    .caption
-                    .copyWith(color: ReplyColors.blue200),
+                style: theme.textTheme.caption.copyWith(
+                  color: ReplyColors.blue200,
```

Comment:
Is this color good for both light and dark mode?

Reply:
Good catch, i'll update it.